Mosaicstg/climate-united
# Use catch-all route

# Description

The following changes have been made:

- Major updates:
   - Remove routes like `/our-people.tsx`, `/case-studies.tsx`, etc
   - Update `$slug.tsx` to ensure it handles finding all content (with a `slug` field) 

- Patch fixes
   - Add `autocomplete` attribute to `<NewsletterSignUp />`
   - Use Remix's `Layout` export in `root.tsx`
   - Add back in the `<LiveReload />` Remix component that was lost in the initial attempt to migrate to Vite

## Type of change


# How Has This Been Tested?

Below are the instructions for testing these updates:

- Update `.env` file to connect to the `master` environment
- Click around ALL pages (of all types) to verify everything is working as expected

# Checklist:
